sui-execution/latest/sui-adapter/src/static_programmable_transactions/mod.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
I am a little confused about the naming -- in what sense is this `static`? Not trying to 🚲 🏠  too hard, just understand the idea.